code/datums/jobs/squads.dm

```diff
@@ -187,10 +187,9 @@
 	if(new_squaddie.assigned_squad)
 		CRASH("attempted to insert marine [new_squaddie] into squad while already having one")
 
-	if(!(new_squaddie.job.title in current_positions))
-		CRASH("Attempted to insert [new_squaddie.job.title] into squad [name]")
-
-	current_positions[new_squaddie.job.title]++
```

Comment:
Deleting this means if someone joins in a job already listed they won't increase current positions, the line below only handles unlisted ones.